specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2022-10-01/machineLearningServices.json

```diff
@@ -4937,6 +5022,126 @@
         "modelAsString": true
       }
     },
+    "TriggerType": {
+      "type": "string",
+      "description": "The schedule trigger type.",
+      "enum": [
+        "Recurrence",
+        "Cron"
+      ],
+      "x-ms-enum": {
+        "name": "TriggerType",
+        "modelAsString": true
+      }
+    },
+    "Recurrence": {
+      "type": "object",
+      "description": "The workflow trigger recurrence for ComputeStartStop schedule type.",
+      "properties": {
+        "frequency": {
+          "description": "[Required] The frequency to trigger schedule.",
+          "$ref": "#/definitions/RecurrenceFrequency"
+        },
+        "interval": {
+          "description": "[Required] Specifies schedule interval in conjunction with frequency",
+          "type": "integer",
+          "format": "int32"
+        },
+        "startTime": {
+          "description": "The start time in yyyy-MM-ddTHH:mm:ss format.",
+          "type": "string",
+          "x-nullable": true
+        },
+        "timeZone": {
+          "description": "Specifies time zone in which the schedule runs.\r\nTimeZone should follow Windows time zone format. Refer: https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/default-time-zones?view=windows-11",
+          "default": "UTC",
+          "type": "string"
+        },
+        "schedule": {
```

Comment:
>schedule [](http://example.com/codeflow?start=9&length=8)

How is this property populated? Does it override the other properties, or is it read-only and derived from them? Or something else?

Reply:
Again, this is something that is coming from the logic apps schema for recurrence. See this [link](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-workflow-actions-triggers#recurrence-trigger) 

The example recurrence below is run every other day, starting from "startTime" at 13:20 hrs.

"recurrence": { 
        "interval": 2, 
        "frequency": "Day", 
        "startTime": "2022-08-11T09:00:00", 
        "timeZone": "India Standard Time", 
        "schedule": { 
            "hours": [13], 
            "minutes": [20] 
        } 
      }